apps/fishing-map/features/map/overlays/rulers/rulers.hooks.ts

```diff
@@ -68,10 +71,11 @@ const useRulers = () => {
   )
 
   const onRulerMapClick = useCallback(
-    (event: MapLayerMouseEvent) => {
+    (info: PickingInfo) => {
+      const [longitude, latitude] = info.coordinate as number[]
```

Comment:
I got this on the first click, could you double check?
<img width="775" alt="image" src="https://github.com/GlobalFishingWatch/frontend/assets/10500650/9b9b12dc-5d73-4682-abbb-e9203ede49a2">